### ISSUE TITLE
Some issue i was found

### ISSUE THREAD

**Issue by khiemledev** [ThemeZee/poseidon#4]
Missing ';' in file inc/template-tags.php, line 408 &laquo => &laquo;
I must to custom these by edit theme

Missing aria-label attribute make page speed insights score is not green: button.search-submit and in .main-navigation-toggle
Set themezee footer text link rel="nofollow noopener" for better score
I must to edit these by writing js
